recipes/default.rb

```diff
@@ -129,9 +125,6 @@ class Chef::Resource # rubocop:disable all
     version node['rabbitmq']['version']
   end
 
-  service node['rabbitmq']['service_name'] do
-    action [:enable, :start]
```

Comment:
other way,  only to enable the service and not start the service, is it more reasonable?   after configuring rabbit, it will be restart immediately now.
@jjasghar